docs/application/toc_all.md

```diff
@@ -51,6 +51,8 @@
 
 ### Applications
 #### [Overview](/application/dotnet/guides/applications/overview.md)
+#### [Tizen .NET Application Lifecycle](/application/dotnet/guides/applications/application_lifecycle.md)
```

Comment:
Now, I am not seeing this topic/file (application_lifecycle.md) in the **files changed** tab in the PR. Please check and include this topic again.